lib/rust/lexer/definition/src/lexer.rs

```diff
@@ -662,8 +1702,8 @@ impl EnsoLexer {
 
         let in_block_line_id = lexer.in_block_line;
         let in_block_line    = lexer.group_mut(in_block_line_id);
-        in_block_line.create_rule(&lf,               "self.block_on_empty_lf_line(reader)");
-        in_block_line.create_rule(&crlf,             "self.block_on_empty_crlf_line(reader)");
+        in_block_line.create_rule(&lf,"self.block_on_empty_line(token::LineEnding::LF,reader)");
+        in_block_line.create_rule(&crlf,"self.block_on_empty_line(token::LineEnding::CRLF,reader)");
```

Comment:
LEts not do this! Lets do what previous lexer was doing. Lets normalize ALL line endings during lexing. We will always save file with removing `\r\n` and replacing it with `\n`. The same applies with tabs - lexer should replace all tabs with 4 spaces (non-configurably)

Reply:
As discussed.